airflow/www/app.py

```diff
@@ -70,8 +70,9 @@ def create_app(config=None, session=None, testing=False, app_name="Airflow"):
     app.json_encoder = AirflowJsonEncoder
 
     csrf.init_app(app)
-
-    db = SQLA(app)
+    db = SQLA()
```

Comment:
This creates a new session, but we want to use your global session instead of creating another session.
https://github.com/pallets/flask-sqlalchemy/blob/706982bb8a096220d29e5cef156950237753d89f/flask_sqlalchemy/__init__.py#L714